spec/lib/phone_number_parser_spec.rb

```diff
@@ -1,23 +1,57 @@
-require "rails_helper"
+require 'rails_helper'
```

Comment:
I am in the habit of using single quotes personally because double quotes theoretically suffer worse performance because Ruby allows interpolation. However I think in practice I have spent longer typing this comment than will ever be saved due to this microoptimization, so I think the larger win is around standardization and there being "a right way" to do it. This is something that we can enforce with a linter.